### DISCUSSION TITLE
Login as an different user

### DISCUSSION THREAD

**Discussion by Colin-988** [rustdesk/rustdesk-server-pro#302] [Q&A]
Hello, is there any way to get remote Access to an device without knowing the password and the other site cant accept the session?

**Comment by rustdesk** [comment]
Do you think it is safe to access a device without knowing the password? Anyone knowing the id can access such a machine, do you think  you would like to use such a system? I know TV has easy-access mode, I guess it has the other mechanism to keep the safety. Sadly, we have no easy-access mode for the time being.
